<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>952</width>
    <height>614</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#MyLoginForm {
background: red;
}

#mainFrame {
border: 3px solid red;
border-radius: 40px;
background: white;
}
QLineEdit {
padding: 1px;
border-style: solid;
border: 2px solid green;
border-radius: 8px;
}
QTextEdit {
padding: 1px;
border-style: solid;
border: 2px solid green;
border-radius: 8px;
}
QDateEdit {
padding: 1px;
border-style: solid;
border: 2px solid green;
border-radius: 8px;
}
QTimeEdit {
padding: 1px;
border-style: solid;
border: 2px solid green;
border-radius: 8px;
}

QPushButton {
color: white;
  background: gray;

border-width: 2px;
border-color: gray;
border-style: solid;
border-radius: 10;
padding: 3px;
font-size: 12px;
padding-left: 5px;
padding-right: 5px;
min-width: 80px;
max-width:1800px;
min-height: 20px;
max-height: 20px;
}
QLabel {
font-weight: blue;
font-size: 13px;
}
QPushButton:hover { background-color: #66c011; }
QComboBox{
border-style: solid;
border: 2px solid green;


 }</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>931</width>
      <height>561</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="ajouter_en">
     <attribute name="title">
      <string>Ajouter Enfant</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_ajouterenfant">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>360</y>
        <width>94</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Ajouter</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_nomenfant">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>80</y>
        <width>124</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_prenomenfant">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>150</y>
        <width>124</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_ageenfant">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>220</y>
        <width>124</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_parent">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>290</y>
        <width>124</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>80</y>
        <width>86</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Nom Enfant :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>150</y>
        <width>100</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Prenom Enfant :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>220</y>
        <width>86</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Age Enfant :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>290</y>
        <width>86</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Parent :</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="afficher_en">
     <attribute name="title">
      <string>Afficher Enfant</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget_enfant">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>80</y>
        <width>691</width>
        <height>371</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_6">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>40</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>470</y>
        <width>94</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Actualiser</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>470</y>
        <width>94</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Supprimer</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>470</y>
        <width>94</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Modifier</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>470</y>
        <width>94</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Exporter</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_triDate">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>470</y>
        <width>94</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Trier par date</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="ajouter">
     <attribute name="title">
      <string>Ajouter Activite</string>
     </attribute>
     <widget class="QLineEdit" name="lineEdit_nom">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>50</y>
        <width>131</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_desc">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>110</y>
        <width>131</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateEdit_datee">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>270</y>
        <width>131</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_ajouter">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>360</y>
        <width>94</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Ajouter</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>174</x>
        <y>50</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Nom Activite :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>110</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Description :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>160</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Duree :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>220</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Temps:</string>
      </property>
     </widget>
     <widget class="QTimeEdit" name="timeEdit_dureeD">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>160</y>
        <width>131</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QTimeEdit" name="timeEdit_dureeA">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>160</y>
        <width>118</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>160</y>
        <width>10</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>270</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Datee :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>330</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Rdv :</string>
      </property>
     </widget>
     <widget class="QTimeEdit" name="timeEdit_temps">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>220</y>
        <width>131</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_rdv">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>310</y>
        <width>131</width>
        <height>87</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_enfant">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>440</y>
        <width>131</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>440</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Enfant :</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="afficher">
     <attribute name="title">
      <string>Afficher Activite</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget_activite">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>711</width>
        <height>351</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="actualiser_activite">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>440</y>
        <width>94</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Actualiser</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_supprimer">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>440</y>
        <width>94</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Supprimer</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_modifier">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>440</y>
        <width>94</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Modifier</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>30</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_pdf">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>440</y>
        <width>94</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Exporter  PDF</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_trier_date">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>440</y>
        <width>94</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Trier par date</string>
      </property>
     </widget>
     <widget class="QPushButton" name="export_exl">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>440</y>
        <width>94</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>exporter EXL</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>952</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
